### PR TITLE
Added database tables and enhanced menu

### PR DESCRIPTION
**Pull request by RobertAndion**

### CODE REVIEW

File: src/Page.py [hunk 1]
```diff
@@ -62,8 +62,18 @@ def __init__(self):
...
+            c = -1
+            print("Welcome to InCollege: *** Where you're no longer going to be broke ***\nAll of our broke students managed to find job!!!"
+                  "\n\n1 - Play Video\n2 - People you may know\n3 - Register\n4 - Login\n5 - Useful Links\n6 - InCollege Important Links\n7 - Exit\nEnter a choice: ")
+            while(True):
+                try:
+                    c = int(input())
+                    if(c in range(0, 8)):
+                        break
+                    else:
+                        print("Please try again with an integer between 0 and 7")
```
Comment (Darvelo00) [inline src/Page.py:74]: can we combine these two problem cases under "Please try again with an integer between 0 and 7"? Feels like we could handle both with one statement

**Review by Darvelo00 — CHANGES_REQUESTED** [review]
looks good, added a minor suggestion if you agree

File: src/Page.py (hunk 1 above)
Comment (Darvelo00) [reply src/Page.py:74]: I meant the else and except

Comment (RobertAndion) [reply src/Page.py:74]: The except will allow an input of 10 for example although we dont have a tenth option, this locks it down further

**Review by Darvelo00 — APPROVED** [review]

**Pull request merged**